client/src/redux/actions/observations.js

```diff
@@ -60,9 +78,26 @@ export const deleteObservations = userSession => async dispatch => {
 };
 
 export const fetchObservations = userSession => async dispatch => {
-  const observations = await getObservations(userSession);
-  const JSONobservations = JSON.parse(observations);
-  if (JSONobservations) {
-    dispatch({ type: FETCH_OBSERVATIONS, observations: JSONobservations });
-  }
+  return userSession.getFile('tempUnit').then(async tempUnit => {
+    dispatch({ type: FETCH_TEMP_UNIT, tempUnit });
```

Comment:
```suggestion
    if (tempUnit) dispatch({ type: FETCH_TEMP_UNIT, tempUnit });
```

Avoid setting `tempUnit` to `null`

Reply:
Bump